presto-docs/src/main/sphinx/language/reserved.rst

```diff
@@ -23,6 +23,7 @@ Keyword                        SQL:2016      SQL-92
 ``CURRENT_DATE``               reserved      reserved
 ``CURRENT_TIME``               reserved      reserved
 ``CURRENT_TIMESTAMP``          reserved      reserved
+``CURRENT_USER``               reserved
```

Comment:
Did not verify if this matches what is actually in standards. Did you?

Reply:
Hello @losipiuk! Looked in sql standard draft 2011 and this appears to have been added after 2002.